scripts/dc2/run_dc2_sprinkler.py

```diff
@@ -1,57 +1,391 @@
+                                                                            glsne_ddf_final,
+                                                                            os.path.join(output_dir, 'host_truth.db'),
+                                                                            return_df=True, overwrite_existing=True)
+
+    lensed_agn_truth = dc2_sprinkler.output_lensed_agn_truth(agn_final_ddf_hosts,
+                                                             agn_final_ddf_lenses,
+                                                             om10_ddf_sprinkler_cat,
+                                                             os.path.join(output_dir, 'lensed_agn_truth.db'),
+                                                             return_df=True, overwrite_existing=True)
+
+    lensed_sne_truth = dc2_sprinkler.output_lensed_sne_truth(sne_final_ddf_hosts,
+                                                             sne_final_ddf_lenses,
+                                                             glsne_ddf_final,
+                                                             os.path.join(output_dir, 'lensed_sne_truth.db'),
+                                                             return_df=True, overwrite_existing=True)
 
-def main(input_dir, agn_input, sne_input, dc2_input,
-         output_dir, agn_output, sne_output, lens_output, host_output):
-
-    agn_cat = os.path.join(input_dir, agn_input)
-    sne_cat = os.path.join(input_dir, sne_input)
-    dc2_cat = os.path.join(input_dir, dc2_input)
-
-    dc2_sprinkler = DC2Sprinkler(input_dir, agn_cat, OM10Reader,
-                                 sne_cat, GoldsteinSNeCatReader,
-                                 dc2_cat, DC2Reader)
-
-    agn_hosts, agn_systems, sne_hosts, sne_systems = \
-        dc2_sprinkler.generate_matched_catalogs()
-
-    print('Writing AGN')
-    agn_output_path = os.path.join(output_dir, agn_output)
-    dc2_sprinkler.output_lensed_agn_truth(agn_hosts, agn_systems,
-                                          agn_output_path, id_offset=0)
-    print('Writing SNe')
-    sne_output_path = os.path.join(output_dir, sne_output)
-    dc2_sprinkler.output_lensed_sne_truth(sne_hosts, sne_systems,
-                                          sne_output_path, id_offset=2000)
-    print('Writing Lenses')
-    lens_output_path = os.path.join(output_dir, lens_output)
-    dc2_sprinkler.output_lens_galaxy_truth(agn_hosts, agn_systems, sne_hosts,
-                                           sne_systems, lens_output_path)
-    print('Writing Hosts')
-    host_output_path = os.path.join(output_dir, host_output)
-    dc2_sprinkler.output_host_galaxy_truth(agn_hosts, agn_systems, sne_hosts,
-                                           sne_systems, host_output_path)
 
 if __name__ == '__main__':
 
+    catalog_version = 'cosmoDC2_v1.1.4_image_addon_knots'
+    agn_db = '/global/cscratch1/sd/jchiang8/desc/sims_GCRCatSimInterface/work/2020-02-14/agn_cosmoDC2_v1.1.4.db'
```

Comment:
I've copied the production location of this file here:
`/global/cfs/cdirs/lsst/projecta/lsst/groups/SSim/DC2/cosmoDC2_v1.1.4/agn_cosmoDC2_v1.1.4.db`